### DISCUSSION TITLE
MAINTAINERS

### DISCUSSION THREAD

**Discussion by KalleHallden** [EXERLOG/exer_log#128] [Ideas]
Hi! Thought I'd open a discussion where we can discuss maintainers. I haven't worked with this before so I do not know how to add maintainers but it seems like I need to create an organization for the project and then that way I should be able to add maintainers. Anyone who has experience with this who can give some advice?
Second is I'm not sure how we should go about figuring out who becomes a maintainer. Maybe not that many people want to so that anyone who does want to can. Or lots of people want to and then it might make sense to have a system?

**Comment by jorre127** [comment]
@KalleHallden Haven't worked with maintainers either so don't know much about them. But I'm assuming they get more rights to change things than people who aren't. in that case I wouldn't allow anyone who wants to be a maintainer to become a maintainer. But definitely have a system in place to choose the maintainers. I guess just choose people that you see have knowledge about building an app, and have contributed a lot already, I would choose someone like @dinurymomshad, assuming he wants to do it as well.

**Reply by momshaddinury** [reply]
No worries. I'd gladly help out. But I'd also vote @jorre127 since he has been spotting bugs in PRs.

**Comment by KalleHallden** [comment]
@dinurymomshad and @jorre127 you have both now been added as maintainers! Just want to say thank you to you both for helping out as much as you have I really appreciate it!

**Comment by momshaddinury** [comment]
@KalleHallden Organisation doesn't have a project. Can we move the old project board here?

**Comment by tenshiAMD** [comment]
@KalleHallden I'm willing to help too, may I join?

**Comment by momshaddinury** [comment]
@KalleHallden if @suzanpradhan agrees, we can add him as a maintainer, in charge of UI.

**Reply by suzanpradhan** [reply]
@dinurymomshad Yeah sure.

**Comment by KalleHallden** [comment]
@dinurymomshad what has happened to the project board we were using before? I can't find it now. But yeah we could move it to the organization, but is that the best place for it? It looks to me like people won't see it if they are just in the repo, only if they are looking at the organization?

@tenshiAMD yeah I'll add you as a maintainer :)

**Reply by momshaddinury** [reply]
People won't really take a look at the board. But when it's linked to issues, they will. So it's better if we move the old board to organisation. 

And since the repository is under organisation, better to have everything related to it under organisation

**Reply by KalleHallden** [reply]
Okay, but the board no longer exists? Or is it just for me?

**Reply by jorre127** [reply]
No it's also gone for me, do we just have to create a new one? Or do you know a way to move the old one @dinurymomshad  ?

**Reply by momshaddinury** [reply]
<img width="946" alt="Screenshot 2022-08-24 at 8 18 59 PM" src="https://user-images.githubusercontent.com/26508767/186442527-8c27b8b7-cd3f-4e1e-9cd8-23f768eca9b4.png">

**Comment by mfederowicz** [comment]
Hi anybody here, what is the status of project, I made 3 PR and progress slow down :( @jorre127 and @tenshiAMD can you look at them. #202, #203, #204 and #205 - are closed by stale bot. Any sugestion how to extend lifetime of selected PR if there are nobody to accept them (I dont want to waste work and delete them)?

**Comment by tenshiAMD** [comment]
> Hi anybody here, what is the status of project

@mfederowicz not sure about that and seems like are not active anymore. Maybe ask @KalleHallden if he is still interested.

> can you look at them. #202, #203, #204 and #205 - are closed by stale bot. 

I reopened and pinned those PR now. Please check.

> Any sugestion how to extend lifetime of selected PR if there are nobody to accept them (I dont want to waste work and delete them)?

To address this one, I think we can ignore closing PRs.

**Reply by mfederowicz** [reply]
Hi @KalleHallden, @tenshiAMD, @jorre127   is everything is ok, i dont notice any activity on this repo for almost 21 days :(
